renpy/exports.py

```diff
@@ -926,7 +929,13 @@ def input(prompt, default='', allow=None, exclude='{}', length=None, with_none=N
     if fixed:
         renpy.ui.saybehavior()
 
-    rv = renpy.ui.interact(mouse='prompt', type="input", roll_forward=roll_forward)
+    rv = None
+    if renpy.emscripten:
+        # escape quotes in prompt and default for javascript usage
+        rv = emscripten.run_script_string("(function callMobilePrompt() { if (window.ontouchstart !== undefined) { return prompt(%r, %r); } return ''})()" % (prompt, default))
```

Comment:
`function callMobilePrompt()` can be replaced with `()=>`, using arrow functions makes the code shorter.